webdriver/tests/bidi/emulation/set_locale_override/conftest.py

```diff
@@ -55,3 +55,102 @@ def another_locale(default_locale, some_locale):
     raise Exception(
         f"Unexpectedly could not find locale different from the default {default_locale} and {some_locale}"
     )
+
+
+@pytest_asyncio.fixture
+async def get_current_navigator_language(bidi_session):
+    async def get_navigator_language(context, sandbox=None):
```

Comment:
nit: we usually keep the same name for the internal function defined in our fixtures (or prefixed with `_`
```suggestion
async def get_current_navigator_language(bidi_session):
    async def get_current_navigator_language(context, sandbox=None):
```